usr/share/rear/layout/prepare/GNU/Linux/210_load_multipath.sh

```diff
@@ -37,8 +37,53 @@ blacklist {
     modprobe dm-multipath >&2
     multipath >&2
```

Comment:
The 
<pre>
        LogPrint "multipath activated"
        dmsetup ls --target multipath
        multipath -l
</pre>
after wthe while loop is 'done' looks wrong in case of
"Multipath is not needed. Continue recovery."
because then at least the "multipath activated" message
is wrong and I also think no further multipath commands
need to be run if multipath is not needed.

I think I would do it like (untested excerpts):
<pre>
    LogPrint "Activating multipath"
    modprobe dm-multipath >&2
    # Asking to the User what to do after multipath command return 1.
    # It could be because no multipath device were found (sles11/rhel6)
    # or a real problem in the multipath configuration.
    prompt="Failed to activate multipath, or no multipath device found."
    rear_workflow="rear $WORKFLOW"
    unset choices
    choices[0]="Multipath is not needed. Continue recovery."
    choices[1]="Run multipath with debug options."
    choices[2]="Enter into rear-shell to manually debug multipath."
    choices[3]="Abort '$rear_workflow'"
    choice=""
    wilful_input=""
    # looping on this menu while multipath failed to list device.
    # Note: On sles11/rhel6, multipath failed if no multipath device is found.
    while ! multipath ; do
        choice="$( UserInput -p "$prompt" -D 0 "${choices[@]}" )" && wilful_input="yes" || wilful_input="no"
        case "$choice" in
            ...
        esac
    LogPrint "Retrying to activate multipath"
    dmsetup ls --target multipath
    multipath -l
done
...
</pre>
but then there is still that create_multipath() function
at the end even if "Multipath is not needed" so that
probably a return from that script is better
if "Multipath is not needed" like (excerpt)
<pre>
        case "$choice" in
            (${choices[0]})
            ...
            return 0
</pre>
i.e. 'return 0' instead of 'break' ?